hardhat.config.linea.ts

```diff
@@ -0,0 +1,35 @@
+import "@nomicfoundation/hardhat-foundry";
+import "@nomicfoundation/hardhat-toolbox-viem";
+import "@nomicfoundation/hardhat-viem";
+import "dotenv/config";
+import "hardhat-deploy";
+import { HardhatUserConfig } from "hardhat/config";
+import baseConfig from "./hardhat.config";
+import "./tasks";
+import {
+    LINEA_EZETH_182DAY,
+    LINEA_EZETH_COORDINATOR,
+    LINEA_FACTORY,
+} from "./tasks/deploy/config/linea";
+
+const { env } = process;
+
+const config: HardhatUserConfig = {
+    ...baseConfig,
+    networks: {
+        linea: {
+            live: true,
+            url: env.HYPERDRIVE_ETHEREUM_URL!,
```

Comment:
for another pr but feel like we should start using the chain-specific url env variables in place of `HYPERDRIVE_ETHEREUM_URL` wdyt?

Reply:
I’m not sure. It’s pretty easy to accidentally deploy to mainnet when you’re testing if we do something like that. For right now, I’d prefer to keep it to this since it’s pretty easy to remember to update it when you’re testing deployments.

Reply:
Ideally we would have a better solution for that, and then I’m all for using the existing environment variables.